src/components/formFields/select/select.tsx

```diff
@@ -16,6 +16,7 @@ interface ISelectProps {
   disabled?: boolean;
   isClearable?: boolean;
   hideErrorMessage?: boolean;
+  menuPlacement?: "auto" | "bottom" | "top";

```

Comment:
`react-select` exports its own type for `menuPlacement`: `MenuPlacement`. So I think, instead of re-writing the type here, we should import it and then our prop would become:

```
menuPlacement?: MenuPlacement;
```